src/averageAges.js

```diff
@@ -15,11 +15,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = people.filter(person => (
+    person.sex === 'm' && (century
+      ? Math.ceil(person.died / 100) === century
+      : true)
+  ));
+
+  const menAverageAge = men.map(man => man.died - man.born);
```

Comment:
It's not `menAverageAge`, check variable name
Here is stored the set of mens` ages only